library/src/test/java/com/datatorrent/lib/helper/OperatorContextTestHelper.java

```diff
@@ -44,22 +42,26 @@ protected DateFormat initialValue()
 
   };
 
-  public static class TestIdOperatorContext extends TestContext implements OperatorContext
+  public abstract static class TestIdOperatorContext extends TestContext implements OperatorContext
```

Comment:
I changed the class name to MockOperatorContext.

I disagree that test classes should use only the OperatorContext interface. They don't do it now and I just gave you a reason why it may be useful to keep it that way. I don't see how that promotes incorrect usage. The OperatorContext interface is for the operator, not for the test code of an operator. In any case, this falls outside of the scope of this PR and can be further discussed separately.